tests/Microsoft.Identity.Test.Unit.net45/PublicApiTests/ConfidentialClientApplicationTests.cs

```diff
@@ -64,7 +64,8 @@ public void MockConfidentialClientApplication_AcquireToken()
                 {
                     "scope1",
                     "scope2"
-                });
+                },
+                new Guid().ToString());
```

Comment:
Guid.NewGuid().   We should also always specify the format of the guid as we expect it.  Due to differences in how ToString() works on different platforms (some require IFormatProvider), there's a AsMatsCorrelationId() extension method in Microsoft.Identity.Client.TelemetryCore.Internal.

Reply:
hmm, still not sure I get this. 

---
In reply to: [297843566](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1235#discussion_r297843566) [](ancestors = 297843566)